### ISSUE TITLE
add static variable for folder "templates"

### ISSUE THREAD

**Issue by nanodocumet** [zombor/KOstache#11]
In this line:
https://github.com/zombor/KOstache/blob/3.1%2Fdevelop/classes/kohana/kostache.php#L226
the value of the folder "templates" is hard-coded. Can it be possible to have that value in a public static variable defined by default with "templates" so users can change accordingly if they want?  I kind of like "views" better than "templates".

```
 $file = Kohana::find_file(Kostache::$directory, $path, 'mustache');
```

thanks.

Nano.

**Comment by shadowhand** [comment]
Walrus had this, I think it is a good idea.

**Comment by shadowhand** [comment]
Allow changing the template directory and extension through Kostache::$template and Kostache::$extension, closed by 341483979d7456159aca10eba3b2ea406ae68535
